amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -110,6 +117,14 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
     };
   };
 
+  getSanitizedURLParams = (searchTerm: string, pageIndex: number, selectedTab: ResourceType) => {
+    const currentTab = this.getSelectedTabByResourceType(selectedTab);
+    this.setState({ selectedTab: currentTab });
```

Comment:
(nit: Url, not URL) 

I'd prefer `setState` not to be called in the `getSanitizedUrlParams` function. The 'getter' function shouldn't be updating the state.

```
const { sanitizedTerm, index, sanitizedTab } = this.getSanitizedURLParams(searchTerm, pageIndex, selectedTab);
this.setState({ selectedTab: sanitizedTab });
this.props.searchAll(sanitizedTerm, ...    )

```